data-raw/northern-ireland-hsct-secondary-care.R

```diff
@@ -0,0 +1,147 @@
+# Northern Ireland - Secondary Care Indicators
+# Health and Social Care Trusts
+
+# ---- Load libs & helpers ----
+library(tidyverse)
+library(geographr)
+library(sf)
+library(healthyr)
+library(demographr)
+library(compositr)
+library(ggridges)
+library(devtools)
+library(readODS)
+
+hsct <- boundaries_trusts_ni18 |>
+  st_drop_geometry()
+
+lookup_trusts <- geographr::boundaries_trusts_ni18 |>
```

Comment:
You can remove the call to geographr here, as it is already loaded in the library calls above.